content/blog/2021/06/kwctl-intro-for-kubernetes-administrators.md

```diff
@@ -0,0 +1,307 @@
+---
+title: Introducing kwctl to Kubernetes Administrators
+authors:
+    - Flavio Castelli
+date: 2021-06-08
+---
+
+We are pleased to announce the availability of a new tool within the Kubewarden
+project: [kwctl](https://github.com/kubewarden/kwctl).
+
+kwctl is a command line utility designed to help both policy authors
+and Kubernetes administrators.
+
+This blog post focuses on the user experience of Kubernetes administrators,
+future ones will cover the policy developer side of the story.
+
+## A real world example: controlling container capabilities
+
+The main character of today's story is Alice. Alice is a Kubernetes
+administrator who wants to keep her Kubernetes cluster secure.
+
+Alice uses many solutions to accomplish that, including
+[Kubernetes Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/).
+
+Pod Security Policies are currently deprecated and are going to be removed from Kubernetes 1.25.
+Due to that, Alice wants to find an alternative to Kubernetes PSP.
+
+Today Alice will work to replace the [container capabilities PSP](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-capabilities-for-a-container)
+with a Kubewarden policy.
+
+We will tag along with Alice, watch all the steps she has to perform
+and learn how `kwctl` will help her.
+
+## Finding the right policy
+
+As a first step, Alice visits [Kubewarden’s Policy Hub](https://hub.kubewarden.io),
+the place where Kubewarden policies can be made discoverable.
+
+Alice enters the *"capabilities"* query and finds a policy that seems to be
+doing exactly what she's looking for. The name of the policy is
+*"psp-capabilities"*.
+
+## Obtaining the policy
+
+Next Alice has to obtain the policy. This can be done with a
+simple command:
+
+```shell
+$ kwctl pull registry://ghcr.io/kubewarden/policies/psp-capabilities:v0.1.4
```

Comment:
Maybe not relevant for this blog post, but more generally: How can I use a private registry for policies and how can `kwctl` help me using it ?

Reply:
Ah, it's further down in the blog post.

Maybe briefly mention it here ?